<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientSQL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>60</width>
    <height>80</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="TreeWidget" name="treeWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <column>
       <property name="text">
        <string notr="true">DataBases</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>db.png</normaloff>db.png</iconset>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionSave_Database"/>
    <addaction name="actionOpen_Query"/>
    <addaction name="actionSave_Query"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionFind_and_Replace"/>
   </widget>
   <widget class="QMenu" name="menuInstruments">
    <property name="title">
     <string>Instruments</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEditor"/>
   <addaction name="menuInstruments"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/addDB.png</normaloff>:/images/addDB.png</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Database">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/save-database.png</normaloff>:/images/save-database.png</iconset>
   </property>
   <property name="text">
    <string>Save Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Query">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/query-icon.jpg</normaloff>:/images/query-icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Open Query</string>
   </property>
   <property name="toolTip">
    <string>Open Query</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_Query">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Query</string>
   </property>
   <property name="toolTip">
    <string>Save Query</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionFind_and_Replace">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/find_replace.png</normaloff>:/images/find_replace.png</iconset>
   </property>
   <property name="text">
    <string>Find and Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/run.png</normaloff>:/images/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/preference-icon-17.png</normaloff>:/images/preference-icon-17.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/manual.png</normaloff>:/images/manual.png</iconset>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/Sign-Info-icon.png</normaloff>:/images/Sign-Info-icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">treewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
